runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -38,3 +38,17 @@ J9::ARM64::TreeEvaluator::generateFillInDataBlockSequenceForUnresolvedField(TR::
    {
    TR_ASSERT_FATAL(false, "This helper implements platform specific code for Fieldwatch, which is currently not supported on ARM64 platforms.\n");
    }
+
+TR::Register *
+J9::ARM64::TreeEvaluator::monentEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_J9VMBase *fej9 = (TR_J9VMBase *) (cg->fe());
+   int32_t lwOffset = fej9->getByteOffsetToLockword((TR_OpaqueClassBlock *) cg->getMonClass(node));
+   TR_ASSERT(lwOffset>=LOWER_IMMED && lwOffset<=UPPER_IMMED, "Need re-work on using lwOffset.");
```

Comment:
Are these 3 lines needed if you're punting to the VM?  If so, what do you have in mind?  If not, please remove.

Reply:
Probably not, I wasn't sure if the assert was important (the other evaluators with lwOffset don't have this check) but otherwise lwOffset is not used for any of the evaluators that will be punting to the VM. I'll remove those lines.